report_webkit_barcode/helper_barcode.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```

Comment:
Short headers like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py